src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { HEADER_HEIGHT } from '../header/header';
+import { Link } from '../links/links';
+import { useI18next } from 'gatsby-plugin-react-i18next';
+import { BreadcrumbText } from '../typography/typography';
+import { theme } from '../layout/theme';
+
+const BreadcrumbContainer = styled.ol<{ hasHero: boolean }>`
+  margin: calc(${HEADER_HEIGHT} + 16px) 24px 0;
+  margin-top: ${(props) => props.hasHero && '16px'};
```

Comment:
I feel like these are values that the breadcrumb component does not need to know. Can't we have the breadcrumb just as a simple UI element that is being placed somewhere on the page? The spacing from the top is something that a parent component (`Layout`?) should take care of.